website/content/docs/concepts/domain-model/credential-libraries.mdx

```diff
@@ -7,39 +7,70 @@ description: |-
-  [credential store][].
+- `name` - If you set this attribute, the `name` must be unique within the credential library's parent [credential store][].
 
-- `description` - (optional)
+- `description` - A user-defined description of the credential library for identification purposes.
 
-### Vault Credential Library Attributes
+### Vault generic credential library attributes
 
-A Vault credential library has the following additional attributes:
+The generic Vault credential library has the following additional attributes:
 
-- `path` - (required)
-  The path in Vault to request credentials from.
+- `path` - (required) The path in Vault to request credentials from.
 
-- `http_method` - (optional: defaults to `GET`)
-  The HTTP method the library uses when requesting credentials from Vault.
-  Can be either `GET` or `POST`.
+- `http_method` - The HTTP method the library uses when requesting credentials from Vault.
+Can be either `GET` or `POST`.
+The default value is `GET`.
 
-- `http_request_body` - (optional)
-  The body of the HTTP request the library sends to Vault when requesting credentials.
-  Only valid if `http_method` is set to `POST`.
+- `http_request_body` - The body of the HTTP request the library sends to Vault when requesting credentials.
+Only valid if `http_method` is set to `POST`.
 
-### Vault Credential Library Parameter Templating
+### Vault SSH certificate credential library attributes
 
-In some cases it can be useful to provide information about a Boundary user or account when making a call to Vault. For example, this can allow picking the correct role when asking for database credentials (if roles are separated per-user), or providing a value to encode in an X.509 certificate generated by Vault. As of Boundary 0.11.1, you can template user and account information into either the path in Vault, the `POST` request body, or both.
+As of Boundary 0.12.0, you can configure SSH credential injection using Vault.
+SSH certificate-based authentication extends key-based authentication using digital signatures.
+Your users' authenticity is determined by a certificate signed by a trusted certificate authority (CA).
+You can configure [Vault's SSH secrets engine](/vault/docs/secrets/ssh) to act as the CA.
+
+SSH certificates let you specify how long they are valid for, who can gain access to a target host, how users can log in, and what commands can be used on the target machine.
+Unlike SSH key pairs, SSH certificates are short-lived and self-destructive.
+
+A Vault SSH certificate credential library has the following additional attributes:
+
+- `path` - (required) The path in Vault to request credentials from.
+
+- `username` - (required) The username to use with the SSH certificate.
+You can create a template for this value using [Vault credential library parameter templating](#vault-credential-library-parameter-templating).
+
+- `key_type` - The type of key to use for the generated SSH private key.
```

Comment:
I think to be consistent with other pages, each of these optional attributes should have `(optional)` before the description.